src/webgpu/api/operation/shader_module/compilation_info.spec.ts

```diff
@@ -194,11 +205,19 @@ g.test('line_number_and_position')
           "GPUCompilationMessages that don't report a line number should not report a line position."
         );
 
-        if (message.lineNum === 0 || message.lineNum === _errorLine) {
+        if (message.lineNum === 0) {
           foundAppropriateError = true;
+          break;
+        }
 
-          // Various backends may choose to report the error at different positions within the line,
-          // so it's difficult to meaningfully validate them.
+        if (message.lineNum === _errorLine) {
+          foundAppropriateError = true;
+          if (_errorLinePos !== null && _errorLinePos !== undefined) {
```

Comment:
nit: it shouldn't be possible for it to be null here (typescript inferred type is `number | undefined` according to vscode)
```suggestion
          if (_errorLinePos !== undefined) {
```